src/blueprint.zig

```diff
@@ -119,7 +123,8 @@ pub const Blueprint = struct {
 					const worldZ = startZ +% @as(i32, @intCast(z));
 
 					const block = self.blocks.get(x, y, z);
-					_ = main.server.world.?.updateBlock(worldX, worldY, worldZ, block);
+					if(flags.preserveVoid or block.typ != voidTyp)
```

Comment:
The flag that's more likely to be true should come first in `or`.

Reply:
I don't understand, it is first o.O

Reply:
Aren't most blocks non-void? Whereas preserveVoid is more 50/50?

Reply:
Hmm, yeah